10up/classifai
# Added "Classify Post" button to process existing content.

### Description of the Change
This PR is a continuation of PR #268 (derives some code from PR #268) and implements the requested functionality in #202, it adds support for the process content of the existing posts. It adds the "Classify Post" button in the ClassifAI sidebar panel. The button loads conditionally, it displays only when `Process content on update` is disabled. Please check the attached video for more details on this.

https://user-images.githubusercontent.com/10613171/190631497-e3f35833-5a21-4d7b-a4c2-0c800574090b.mp4

Closes #202 

### How to test the Change
1. Create a few posts without activating the ClassifAI plugin.
2. Activate the ClaissifAI plugin and language processing.
3. Go to edit post page and open the ClassifAI sidebar panel.
4. Disable the `Process content on update` option
5. Make sure a "Classify Post" button is visible now. (`Post` text will be changed for the post type. for Page, it will be "Classify Page")
6. Click it, Verify that terms are generated properly for each taxonomy.
7. Test "Classify Post" button for the classic editor as well. 

### Changelog Entry

> Added - "Classify Post" button to process existing content.


### Credits
Props @thrijith @iamdharmesh @dkotter @dinhtungdu @jeffpaul 


### Checklist:
